godot-cell/src/borrow_state.rs

```diff
@@ -38,6 +42,8 @@ impl BorrowState {
             mut_count: 0,
             inaccessible_count: 0,
             poisoned: false,
+            #[cfg(feature = "threads")]
+            thread_shared_count: Default::default(),
```

Comment:
```suggestion
            thread_shared_count: HashMap::new(),
```
is a bit clearer.

Regarding the conditional import, I'm fine with both fully qualifying it, or adding a `#[cfg]` on the `use` statement.